CMakeLists.txt

```diff
@@ -124,11 +124,72 @@ target_sources(lcf
 	src/generated/lsd_savetarget.cpp
 	src/generated/lsd_savetitle.cpp
 	src/generated/lsd_savevehiclelocation.cpp
+	src/generated/rpg_actor.cpp
+	src/generated/rpg_animation.cpp
+	src/generated/rpg_animationcelldata.cpp
+	src/generated/rpg_animationframe.cpp
+	src/generated/rpg_animationtiming.cpp
+	src/generated/rpg_attribute.cpp
+	src/generated/rpg_battlecommand.cpp
+	src/generated/rpg_battlecommands.cpp
+	src/generated/rpg_battleranimation.cpp
+	src/generated/rpg_battleranimationdata.cpp
+	src/generated/rpg_battleranimationextension.cpp
 	src/generated/rpg_chipset.cpp
-	src/generated/rpg_enums.cpp
```

Comment:
gone?

Reply:
Really good catch. This would only manifest as a linker error for debug builds on some platforms. Fixed.